src/utils/dataAttributes.ts

```diff
@@ -0,0 +1,32 @@
+export const DataAttribute = {
+  ElementCodename: 'data-kk-element-codename',
+  ItemId: 'data-kk-item-id',
+  LanguageCodename: 'data-kk-language-codename',
+  ProjectId: 'data-kk-project-id',
+  // TODO: [NOT FOR ETP] add attributes for page-item-id and page-language-codename.
+  // TODO: [NOT FOR ETP] maybe add some attribute to show that item does not have elements inside and must be processed as a node with data-kk-element-codename attribute.
```

Comment:
These TODOs should rather be in a follow-up task rather than in the release code